### DISCUSSION TITLE
Portable standalone package availability

### DISCUSSION THREAD

**Discussion by mmalgrange** [Ifsttar/I-Simpa#302] [General]
Hi, 
Currently, the portable standalone package link refers to an error.
{"message":"Artifact not found or access denied."}
Is this a bug or is this version no longer maintained ?
Thanks for your answer

**Comment by nicolas-f** [comment]
Hi,

Where do you find this message ?

**Comment by mmalgrange** [comment]
Hi, 

I find this message on this page : 
https://i-simpa.univ-gustave-eiffel.fr/download/download0/
by clicking on the [**Portable standalone package**](https://ci.appveyor.com/api/projects/nicolas-f/I-Simpa/artifacts/build%2Fisimpa-1.3.4-win64.zip?branch=master) link.

Thanks

**Comment by mmalgrange** [comment]
Hi, 

did you have the time to look at this point ?
Thanks in advance

**Comment by Picaut** [comment]
@nicolas-f  could you give the good link. I will update the website

**Reply by nicolas-f** [reply]
https://github.com/Ifsttar/I-Simpa/releases/download/v1.3.4/isimpa-1.3.4-win64.zip
